fast/stages/2-project-factory/README.md

```diff
@@ -0,0 +1,354 @@
+
+The pattern implemented here by default allows management of a teams (or business units, applications, etc.) hierarchy. Different patterns are possible, and this document also tries to provide some guidance on how to implement them.
+
+<p align="center">
+  <img src="diagram.png" alt="Project factory teams pattern">
+</p>
+
+## Design overview and choices
+
+The project factory is "primed" by the resource management stage via
+
+- a set of service accounts with different scopes
+- one or more user-defined top-level folders where those service accounts operate
+
+This stage does not directly depend from other stage 2 like networking and security, but it can optionally leverage resources created there like Shared VPC host projects, which are used to define service projects.
+
+The project factory stage lightly wraps the underlying [project-factory module](../../../modules/project-factory/), which in turn exposes the full interface of the [project](../../../modules/project/) and [folder](../../../modules/folder/) modules.
+
+## How to run this stage
+
+This stage is meant to be executed after the [bootstrap](../0-bootstrap/) and [resource management](../1-resman/) "foundational stages". As mentioned above it runs in parallel with other stage 2 and can leverage resources they create but does not depend on them.
+
+### Resource Management stage configuration
+
+The resource management stage already contains a sample "Teams" folder defined via YAML, which can be used as-is or modified to provide a top-level folder for the project factory. More folders can of course be added, and Terraform variables used instead of or in addition to YAML files in the resource management stage.
+
+This is the teams YAML in resource management, leveraging attribute substitutions for the project factory service account and tag value.
+
+```yaml
+name: Teams
+automation:
+  enable: false
+iam:
+  "roles/owner":
+    - project-factory
+  "roles/resourcemanager.folderAdmin":
+    - project-factory
+  "roles/resourcemanager.projectCreator":
+    - project-factory
+  "service_project_network_admin":
+    - project-factory
+tag_bindings:
+  context: context/project-factory
+```
+
+This is the alternative version that can be used instead of the YAML file above.
+
+```tfvars
+top_level_folders = {
+  # more top-level folders might be present here
+  teams = {
+    name = "Teams"
+    iam = {
+      "roles/owner"                          = ["project-factory"]
+      "roles/resourcemanager.folderAdmin"    = ["project-factory"]
+      "roles/resourcemanager.projectCreator" = ["project-factory"]
+      "service_project_network_admin"        = ["project-factory"]
+    }
+    tag_bindings = {
+      context = "context/project-factory"
+    }
+  }
+}
+# tftest skip
```

Comment:
this would be a nice example to test

Reply:
![image](https://github.com/user-attachments/assets/1b67da53-b00f-4353-aebf-6439f17c2723)

we're already testing the YAML equivalent

Reply:
same as above comment, the YAMl version is already part of resman test